### PR TITLE
feat(live-sensors): 15603 frontend shouldnt duplicate live sensor data

### PR DESCRIPTION
**Pull request by Akiyamka**
- Added https mode for dev server - usefull for testing live_sensor feature because it works only in secured env
- Logic rewritten from atoms to pure classes, reactivity not needed here
- Tests added
- Fixed few additional bugs like - button still pressed if user disallow sensor usage, dublicate data sended
- Unique id added to every snapshot (every feature)
- Docs updated

### CODE REVIEW

File: src/features/live_sensor/SensorsSnapshotsSender.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  }
+
+  public start() {
+    this.sendFromQueue();
+  }
+
+  private running = true;
+  async sendFromQueue() {
+    // * Note: will not work in concurrent mode
+    while (this.running && this.sensorsRecords[0]) {
```
Comment (vkozio) [inline src/features/live_sensor/SensorsSnapshotsSender.ts]: Will it be more clear to check array length?

File: src/features/live_sensor/sensors/AppSensorAccelerometer.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    }
+  }
+
+  public stop() {
+    if (this.sensor) {
+      this.sensor.removeEventListener('reading', this.readingHandler);
+      this.sensor.removeEventListener('error', this.errorHandler);
+      this.sensor.stop();
+    }
+  }
```
Comment (vkozio) [inline src/features/live_sensor/sensors/AppSensorAccelerometer.ts]: it's similar for several sensors - can be moved to parent class

**Review by vkozio — APPROVED** [review]
LGTM, but please check comments

File: src/features/live_sensor/sensors/AppSensorAccelerometer.ts (hunk 2 above)
Comment (Akiyamka) [reply src/features/live_sensor/sensors/AppSensorAccelerometer.ts]: unforutantelly parent class don't known about sensors from typescript perspective

File: src/features/live_sensor/SensorsSnapshotsSender.ts (hunk 1 above)
Comment (Akiyamka) [reply src/features/live_sensor/SensorsSnapshotsSender.ts]: agree

**Pull request merged**